application/templates/deednotfound.html

```diff
@@ -2,6 +2,9 @@
 
 {% block content %}
 <div class="main" id="content">
+
+{{ super() }}
```

Comment:
so all the borrower-frontend template files extend the "base/lr_base_layout.html”, the super keyword allows the block_content to inherit from its parent block in the "base/lr_base_layout.html” which contains the beta banner elements. All template files  use the super function, nice clean way to include the beta banner across the board.